client/src/components/user-admin/UserManagement.js

```diff
@@ -42,69 +49,108 @@ const UserManagement = ({ users, setUserToEdit }) => {
+            mx: '0.5rem'
+            }}>
+              <Button
+                sx={Buttonsx}
+                type="button"
+                variant={
+                  searchResultType === 'name'
+                  ? 'contained'
+                  : 'secondary'
+                }
+                onClick={buttonSwap}
+              >
+                Results by Name
+              </Button>
+              <Button
+                sx={Buttonsx}
+                type="button"
+                variant={
+                  searchResultType === 'email'
+                  ? 'contained'
+                  : 'secondary'
+                }
+                onClick={buttonSwap}
+              >
+                Results by Email
+              </Button>
+          </ButtonGroup>
+        </Box>
+        <TextField
           type="text"
-          placeholder="Search by name and email..."
+          placeholder="Enter name and / or email to find a user."
+          variant='standard'
           value={searchTerm}
           onChange={handleChange}
         />
-        <div className="tab-buttons">
-          <div>
-            <button
-              type="button"
-              className={
-                searchResultType === 'name'
-                  ? 'select-button selected'
-                  : 'select-button'
-              }
-              onClick={buttonSwap}
-              disabled={searchResultType === 'name'}
-            >
-              Results by Name
-            </button>
-          </div>
-          <div>
-            <button
-              type="button"
-              className={
-                searchResultType === 'email'
-                  ? 'select-button selected'
-                  : 'select-button'
-              }
-              onClick={buttonSwap}
-              disabled={searchResultType === 'email'}
-            >
-              Results by Email
-            </button>
-          </div>
-        </div>
-        <div>
-          <div>
-            <ul className="search-results">
-              {searchResults.map((u) => {
-                return (
-                  // eslint-disable-next-line no-underscore-dangle
-                  <li key={`result_${u._id}`}>
-                    <button
-                      className="search-results-button"
-                      type="button"
-                      onClick={() => setUserToEdit(u)}
-                    >
-                      {searchResultType === 'name'
-                        ? `${u.name?.firstName} ${u.name?.lastName} ( ${u.email} )`
-                        : `${u.email} ( ${u.name?.firstName} ${u.name?.lastName} )`}
-                    </button>
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
-        </div>
-      </div>
-    </div>
+        <Box sx={{ 
+          bgcolor: searchResults.length>0? '#F5F5F5': 'transparent',
+          my: 1.2,
+          borderRadius: 1,
+          display: 'flex',
```

Comment:
Please drop this prop, it's creating layout shift when the returned results of the user search aren't max length. This change will revert the box to `display: block`

Reply:
dropping the prop `display: 'flex' and flexGrow: 1` results in bringing the button closer to the `search-bar`, whereas in the design they have a significant amount of space.

Reply:
The box doesn't need display flex to have flex grow work

Reply:
Got it ......

Reply:
Please see screenshot:

<details>
<summary>With `display: flex`</summary>
<img width="969" alt="image" src="https://github.com/hackforla/VRMS/assets/5898009/60bc6f49-fba5-420a-8374-47fdea04bcef">
</details>

<details>
<summary>Without `display: flex`</summary>
<img width="969" alt="image" src="https://github.com/hackforla/VRMS/assets/5898009/2db863dd-8d89-4dce-8499-bb24a636cdf4">

</details>